index.html

```diff
@@ -0,0 +1,182 @@
+            <span class="iconify icon-btOverView" data-icon="fa-solid:faucet"> </span>
+            <p>AMOUNT TO PRODUCE</p>
+            <p>6000</p>
+          </div>
+          <div class="acceptableProducts btOV-child">
+            <span class="iconify icon-btOverView" data-icon="bx:bx-check"> </span>
+            <p>ACCEPTABLE PRODUCTS</p>
+            <p>223</p>
+          </div>
+          <div class="vibration btOV-child">
+            <span class="iconify icon-btOverView" data-icon="zmdi:vibration"></span>
+            <p>VIBRATION</p>
+            <p>0.0</p>
+          </div>
+          <div class="productsPerMin marked btOV-child">
+            <span class="iconify icon-btOverView" data-icon="cil:speedometer"> </span>
+            <p>PRODUCTS PER MINUTE</p>
+            <p>300</p>
+          </div>
+          <div class="DefectProducts btOV-child">
+            <span class="iconify icon-btOverView" data-icon="bi:x"> </span>
+            <p>DEFECT PRODUCTS</p>
+            <p>43</p>
+          </div>
+        </div>
+
+        <!-- Inventory of the beer machine -->
+        <div class="inventory-st">
+          <div class="inventory-top">
+            <p>INVENTORY STATUS</p>
+          </div>
+          <div class="inventory-container">
+            <div class="BARLEY inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="BARLEY inventory-overlay">
+                <p>BARLEY</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="MALT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="MALT inventory-overlay">
+                <p>MALT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="HOPS inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="HOPS inventory-overlay">
+                <p>HOPS</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="WHEAT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="WHEAT inventory-overlay">
+                <p>WHEAT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="YEAST inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="YEAST inventory-overlay">
+                <p>YEAST</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- maintain info -->
+        <div class="maintains">
+          <div id="maintains-container">
+            <div></div>
+            <div></div>
+          </div>
+          <div id="maintains-bottom">
+            <p>100%</p>
+            <span class="iconify" data-icon="mdi:wrench"></span>
+          </div>
+        </div>
+
+        <!-- graph view -->
+        <div class="graphs">
```

Comment:
We need to make the text change depending on which button is pressed.

Reply:
yes we can add a class to graph view, so it will be esay to update with javascript.

Reply:
I have made another issus for this, because it is a bit bigger than I would like this update to be
https://github.com/bierproductie/dashboard/issues/2

Reply:
Great

Reply:
Great.